requirements.txt

```diff
@@ -107,7 +107,7 @@ sphinxcontrib-qthelp==2.0.0
     # via sphinx
 sphinxcontrib-serializinghtml==2.0.0
     # via sphinx
-sqlalchemy==2.0.38
+sqlalchemy==1.4.54
```

Comment:
Good find!

Reply:
that was the result of trying to run things a couple times, and being confused as to why i was still getting complaints about incompatible versions, even after updating the `pyproject.toml` file 😄